packages/plugins/FileService/src/Worker/ipfs.ts

```diff
@@ -0,0 +1,70 @@
+import { Attachment } from '@dimensiondev/common-protocols'
+import { encodeText } from '@dimensiondev/kit'
+import { create, IPFSHTTPClient } from 'ipfs-http-client'
+import { isEmpty } from 'lodash-unified'
+import { landing } from '../constants'
+import urlcat from 'urlcat'
+import type { ProviderAgent, LandingPageMetadata, AttachmentOptions } from '../types'
+import { makeFileKeySigned } from '../helpers'
+
+function creatClient(): IPFSHTTPClient {
+    return create({
+        host: 'ipfs.infura.io',
+        port: 5001,
+        protocol: 'https',
+    })
+}
+
+class IPFSAgent implements ProviderAgent {
+    static providerName = 'IPFS'
+    client!: IPFSHTTPClient
+
+    init() {
+        if (!this.client) {
+            this.client = creatClient()
+        }
```

Comment:
```suggestion
        if (this.client) return
        this.client = createClient()
```

`creat` is typo error